components/create-lead/create-lead-form.tsx

```diff
@@ -259,7 +259,7 @@ const CreateLeadForm: React.FC = () => {
       <div className='flex justify-end  mt-5 gap-5 items-end'>
         <Button
           onClick={submitData}
-          disabled={images.length === 0 || pending === true}
+          // disabled={images.length === 0 || pending === true}
```

Comment:
While creating Lead this button should disable, so I think we should keep `pending === true` and remove `images.length === 0`.